src/scripts.js

```diff
@@ -42,36 +42,57 @@ const errorHandlingLine = document.getElementById('handleMyError')
 
 //CLASS INSTANTIATIONS
 let user;
-let bookings;
-let rooms;
 let bookingsData;
 let roomsData;
 let singleUser;
 
 
 //FUNCTIONS
 
-function getData() {
-  return Promise.all([fetchApiData('customers'), fetchApiData('rooms'), fetchApiData('bookings')]);
+function getData(userID) {
+  return Promise.all([fetchSingleUser(userID), fetchApiData('rooms'), fetchApiData('bookings')])
 }
 
+//on click
+const loginUser = (event) => {
+  event.preventDefault();
+  let username = usernameInput.value
+  let userID = username.substring(8)
+  if(userID > 0 && userID < 51 && passwordInput.value === 'overlook2021') {
```

Comment:
Love that you added edge cases in case the user enters a userId that is not part of the data.